src/openepd/api/sync_client.py

```diff
@@ -23,10 +26,18 @@
 from openepd.api.pcr.sync_api import PcrApi
 
 
+class OpenEpdApiClientSyncKwargs(TypedDict):
+    throttle_retry_timeout: NotRequired[float | int | datetime.timedelta]
+    requests_per_sec: NotRequired[float]
+    retry_count: NotRequired[int]
+    user_agent: NotRequired[str | None]
+    timeout_sec: NotRequired[float | tuple[float, float] | None]
+
+
 class OpenEpdApiClientSync:
     """Synchronous API client for OpenEPD."""
 
-    def __init__(self, base_url: str, auth_token: str | None, **kwargs) -> None:
+    def __init__(self, base_url: str, auth_token: str | None, **kwargs: Unpack[OpenEpdApiClientSyncKwargs]) -> None:
```

Comment:
is this supported in python 3.10?

Reply:
Also, we are typing this differently to all other places - is this because we know what's here in kwargs?

Reply:
>is this supported in python 3.10? 

Do we support 3.10 with openepd?